colors/oceanicnext.vim

```diff
@@ -407,6 +407,17 @@ hi! link GitGutterChangeDelete OcOrange
 
 " }}}
 
+" ----- Plugin (vim-signify) --------------------------------------------- {{{
```

Comment:
Could you add two dashes (`--`) before `{{{` to align it with the other sections?

Reply:
@adrian5 I made the requested change.